### ISSUE TITLE
Why Can't I get the mask or boxes?

### ISSUE THREAD

**Issue by MikoyChinese** [youngwanLEE/CenterMask#22]
Hi,

When I run the `configs/caffe2/e2e_mask_rcnn_R_50_FPN_1x_caffe2.yaml`, I can get the mask and boxes. But when I try to use centermask, `configs/centermask/centermask_M_v2_FPN_lite_res600_ms_bs32_1x.yaml`, I can get nothing.

Here are my code:

```
import numpy as np
from maskrcnn_benchmark.config import cfg
from predictor import COCODemo

config_file = "../configs/centermask/centermask_M_v2_FPN_lite_res600_ms_bs32_1x.yaml"

# update the config options with the config file
cfg.merge_from_file(config_file)
# manual override some options
cfg.merge_from_list(["MODEL.DEVICE", "cuda"])

coco_demo = COCODemo(
    cfg,
    confidence_threshold=0.2
)

img = cv2.imread('Test.jpg')
predictions = coco_demo.run_on_opencv_image(color_image)
```

When I show the predictions, I just get the original image.
Can you help me?

Best,
Mikoy Chinese

**Comment by youngwanLEE** [comment]
Did you load the pretrained [weight](https://www.dropbox.com/s/fk9m4uqkhrpkqc6/centermask-lite-M-v2-bs16-4x.pth?dl=1)?

Could you show me the command line you use?

Why don't you use our [demo program](https://github.com/youngwanLEE/CenterMask/blob/master/demo/centermask_demo.py)?

**Comment by MikoyChinese** [comment]
I got the pretrained weight from `COCODemo` which will download a file into ~/.torch/models.

I build this in my docker following your dockerfile, and I found that the dockerfile is base on maskrcnn-benchmark not centermask, so I use pip install detectron2 and clone the centermask and build it again.

The following code remove from `maskrcnn-benchmark/demo/predictor.py`.
```
        if weight_loading:
            print('Loading weight from {}.'.format(weight_loading))
            _ = checkpointer._load_model(torch.load(weight_loading))
```

**Comment by youngwanLEE** [comment]
@MikoyChinese 
I mean the pretrained weight is coco-trained model, [centermask-lite-Mv2 weight]((https://www.dropbox.com/s/fk9m4uqkhrpkqc6/centermask-lite-M-v2-bs16-4x.pth?dl=1)

**Comment by MikoyChinese** [comment]
Thx for your help, I will try your demo first.

**Comment by MikoyChinese** [comment]
Finally, I found the reason, it needs me to input the weight file path with `cfg.MODEL.WEIGHT = './centermask-lite-M-v2-bs16-4x.pth?dl=1'`.

Here are the fixed code:

```
import numpy as np
from maskrcnn_benchmark.config import cfg
from predictor import COCODemo

config_file = "../configs/centermask/centermask_M_v2_FPN_lite_res600_ms_bs32_1x.yaml"

# update the config options with the config file
cfg.merge_from_file(config_file)
# manual override some options
cfg.merge_from_list(["MODEL.DEVICE", "cuda"])
# Fixed it.
cfg.MODEL.WEIGHT = './centermask-lite-M-v2-bs16-4x.pth?dl=1'
cfg.freeze()

coco_demo = COCODemo(
    cfg,
    confidence_threshold=0.2
)

img = cv2.imread('Test.jpg')
predictions = coco_demo.run_on_opencv_image(color_image)
```

Sorry about that I confused the `facebookresearch/maskrcnn-benchmark` and `youngwanLEE/CenterMask`.

Thanks for your help.

Best,
Mikoy Chinese
